src/mod.js

```diff
@@ -0,0 +1,3 @@
+export default function modA () {
```

Comment:
what's this?

Reply:
Oh, that's a leftover test piece. Will remove the file